openml/extensions/sklearn/extension.py

```diff
@@ -743,7 +830,9 @@ def flatten_all(list_):
                 else:
                     parameters[k] = None
 
-            parameters_meta_info[k] = OrderedDict((('description', None), ('data_type', None)))
+            data_type, description = parameters_docs[k]
```

Comment:
Can we make this more failure tolerant if the parameter is not documented? We should then also issue a warning.

Reply:
Have added a if...else which is like communicates an all-or-nothing scenario.